File: 3. Conditional Loops/1. Cpp/2. Intermediate/power.cpp
#include <bits/stdc++.h>
using namespace std;
#define FIO ios_base::sync_with_stdio(false), cin.tie(0), cout.tie(0)
#define ll long long
#define endl "\n"
int main() {
	FIO;
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	int a, b;
	cin >> a >> b;
	int power;
	if (b <= 0) {
		cout << "Power cannot be calculated" << endl;
	}
	else {
		cout << "Power is: " << pow(a, b) << endl;
	}
	return 0;
}

File: 3. Conditional Loops/2. Java/2. Intermediate/power.java
import java.util.Scanner;
// import java.math.*;
public class power {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        double a,b;
        System.out.print("A: ");
        a = sc.nextDouble();
        System.out.print("B: ");
        b = sc.nextDouble();
        double power = Math.pow(a,b);
        if(b<=0){
            System.out.print("Power cannot be calculated");
        }
        else{
            System.out.print("Power is: " + power);
        }
        sc.close();
    }
}
